intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+    if (raw_data.keys.length > 0) {
+        $("#members-chart").show();
+        canvas = $("#members-chart")[0];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["canvas" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2620490638/issues/source?bid=3212528&fileBranchId=3299315#l4)